optimum_benchmark/scenarios/inference/scenario.py

```diff
@@ -61,14 +57,6 @@ def __init__(self, config: InferenceConfig) -> None:
         super().__init__(config)
 
     def run(self, backend: Backend[BackendConfigT]) -> BenchmarkReport:
-        if is_torch_distributed_available() and torch.distributed.is_initialized():
```

Comment:
Great! Works for all TP distribution. not only  pytorchbackend but also vllm and tensorRT.

Reply:
Yeah for those the TP happens in the engine (no need for torchrun laucher).